lib/fizzy/stack.hpp

```diff
@@ -6,31 +6,53 @@
 namespace fizzy
 {
 template <typename T>
-class Stack : public std::vector<T>
+class Stack
 {
+    T storage[1024];
+
+    T* top_item;
```

Comment:
Would be interesting to see the difference when fields order is reversed.

Reply:
Swapped.